examples/tree_2d_dgsem/elixir_navierstokes_taylor_green_vortex_sutherland.jl

```diff
@@ -0,0 +1,91 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Navier-Stokes equations
+
+# TODO: parabolic; unify names of these accessor functions
+prandtl_number() = 0.72
+
+# Use Sutherland's law for a temperature-dependent viscosity
```

Comment:
Can you add a reference or link for Sutherland's law here?

Reply:
Yes, I added Viscous Fluid Flow by White as a reference (without a link, the one I found is rather lengthy).